en/docs/learn/examples/rest-api-examples/securing-rest-apis-with-jwt-access-tokens.md

```
@@ -411,38 +411,38 @@ Follow the instructions below to secure APIs with JWT (Self Contained) access to
-        </resource>
-        <handlers>
-            <handler class="org.wso2.micro.integrator.security.handler.oauth.OAuth2AuthorizationHandler">
-                <property name="jwksEndpoint" value="https://idp.example.com/.well-known/jwks.json"/>
-                <property name="audience" value="https://api.orders.com"/>
-                <property name="authorizationHeader" value="Authorization"/>
-                <property name="disableCNFValidation" value="false"/>
-            </handler>
-        </handlers>
-    </api>
-    ```
+  <img src="{{base_path}}/assets/img/learn/api-security/oauth2/oauth2-authorization-handler/add-oauth-authorization-handler.png">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)